### ISSUE TITLE
Accessible Table using Keyboard

### ISSUE THREAD

**Issue by mghuy** [react-tools/react-table#634]
## What version of React-Table are you using?
Latest Version (v6.7.4)

## What bug are you experiencing, or what feature are you proposing?
Keyboard navigation is currently non-existent to the rows of data that are expandable. While users who can use a mouse can easily access this collapsible feature, those using the tab button for navigation cannot. I plan to resolve this issue by adding tab features to expandable rows in compliance with WAI-ARIA accessibility standards.

**Comment by gary-menzel** [comment]
When you have the update ready, please feel free to submit the PR for review.

**Comment by gary-menzel** [comment]
@mghuy Happy to accept any PR you can prepare in regard to this but will close the issue for now. Please re-open once the PR is ready.
